src/commands/build.rs

```diff
@@ -14,5 +14,25 @@ pub fn run(matches: &ArgMatches) -> Result<(), failure::Error> {
     let manifest = Manifest::new(&config_path)?;
     let env = matches.value_of("env");
     let target = &manifest.get_target(env, false)?;
-    build(&target)
+    if matches.is_present("output") {
```

Comment:
Usually the clap matching stuff is done in main and then an `Option` with the arg is passed in where you can do this check and the one below. Something like
```rust
if let Some(output) = output {
  if output == "json" {
    ....
  }
}
```

Reply:
That makes sense to keep the clap matching in main whenever possible. However, in this case the path for `wrangler build` leaves the matching to this `run` function. I'll do the matching in main and pass an `Option` for the `wrangler publish` path.